### PR TITLE
add Aurora

### PR DESCRIPTION
**Pull request by gruzzlymug**
**Description:**
add aurora to RDS price collection

**Review:**

- [ ] Do the unit tests contain positive functionality tests as well as edge case negative tests?
- [ ] Can we monitor the performance/effectiveness of these changes?
- [ ] Did the code review consider failure modes and edge cases?
- [ ] Did the code review consider refactorings that would make this code more maintainable?
- [ ] Was this change approved by a module owner (if necessary)?

### CODE REVIEW

File: lib/amazon-pricing/rds-price-list.rb [hunk 1]
```diff
@@ -152,21 +163,22 @@ def fetch_reserved_rds_instance_pricing2(url, dbs, is_multi_az, is_byol)
     end
 
     def get_rds_reserved_instance_pricing
-       @@DB_TYPE.each do |db|
+      @@DB_TYPE.each do |db|
+        next if @@DB_RI_PRICE_ONLY_FROM_API.include? db
```
Comment (jeffzhou791) [inline lib/amazon-pricing/rds-price-list.rb]: Did you test that aurora prices are good from json files? If so we don't need to skip aurora

Comment (jeffzhou791) [inline lib/amazon-pricing/rds-price-list.rb]: do you really need this line? also, even if you need to do this, this class variable should be renamed

**Comment by jeffzhou791** [comment]
:+1: with one comment.
